### DISCUSSION TITLE
Can't find or link to hdf5 library

### DISCUSSION THREAD

**Discussion by guangdongzhang** [peridigm/peridigm#242] [Q&A]
When I build netcdf-4, it always show can't find or link to hdf5 library, what's the problem?

**Comment by djlittl** [comment]
The problem is that it can’t find the hdf5 library.  The hdf5 library needs to be built on your system, and the netcdf build system needs to be able to find it.  This is a very basic build error, the type of thing you can copy-and-paste into a Google search.
